includes/wikia/transaction/TransactionTraceNewrelic.php

```diff
@@ -23,7 +23,6 @@ class TransactionTraceNewrelic {
 		'TemplateClassificationService::getType', # Template classification
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)